rules/windows/credential_access_iis_connectionstrings_dumping.toml

```
@@ -0,0 +1,48 @@
+[metadata]
+creation_date = "2020/08/18"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/08/18"
+
+[rule]
+author = ["Elastic"]
+description = """
+An attacker with Microsoft IIS webserver access via a webshell or alike can decrypt and dump any hardcoded
```

```suggestion
An attacker with Microsoft IIS web server access via a webshell or alike can decrypt and dump any hardcoded
```

to be consistent with usage in previous rules